tests/checkout/icase.c

```diff
@@ -100,7 +100,11 @@ static void assert_name_is(const char *expected)
 	char *actual;
 	size_t actual_len, expected_len, start;
 
+	if (!cl_sandbox_supports_8dot3())
+		clar__skip();
```

Comment:
So, this is a little weird since we're not actually doing anything with 8.3 paths.  You're really trying to avoid running this test in places where `GetFinalPathNameByHandle` fails...  I think this problem is distinct from a lack of 8.3 paths, though, isn't it?

If so, we should consider refactoring the tests to skip when `GetFinalPathNameByHandle` is not available.

I say this with great reluctance, because I believe that your system is really broken if that's not supported and that this is a terrible bug in your filesystem that warrants replacement.  That's not hyperbole, I would really consider a different ramdisk driver if random function calls fail, because it makes running tests really difficult if you can't trust them.

Maybe instead of trying to decide whether or not your environment is flaky and we should skip the tests we should have an environment variable here.  That way there is no risk of us misinterpreting an actual test failure as one that would have come from an aforementioned flaky environment.